### PR TITLE
Set click pixel tolerance to 5px, be able to accept click and longpress

### LINKED ISSUE #1201: Difficulty to query with Chrome
After some tests with Chrome on an Android Tablet, I noticed that it's very hard to make a GetFeature, even after update to `1.5.3dev-20150126`.

In fact, when "clicking" on the map, the map moves so the GetFeature is not fired. I tested with the native Android browser, the map doesn't move when clicking on it so it's more much simple to make a query.

I don't know if it's related to C2Cgeoportal or OpenLayers.

@kalbermattenm @gnerred Did you noticed the same behaviour?

### PR DESCRIPTION
**Pull request by sbrunner**
Fix #1201

The in the `<package>/static/mobile/config.js` file `App.queryMode` should be set to "click" or "both".

### CODE REVIEW

File: c2cgeoportal/scaffolds/create/+package+/static/mobile/config.js_tmpl [hunk 1]
```diff
@@ -22,7 +22,7 @@ try {
 
     App.WFSTypes = jsonFormat.read('${dumps(wfs_types) | n}');
 
-    // Query mode. Can be either 'click' or 'longpress'
+    // Query mode. Can be either "click", "longpress", "booth" or "none"
```
Comment (elemoine) [inline c2cgeoportal/scaffolds/create/+package+/static/mobile/config.js_tmpl]: What's "booth"?

Comment (ochriste) [inline c2cgeoportal/scaffolds/create/+package+/static/mobile/config.js_tmpl]: both

Comment (sbrunner) [inline c2cgeoportal/scaffolds/create/+package+/static/mobile/config.js_tmpl]: :-)

Comment (sbrunner) [inline c2cgeoportal/scaffolds/create/+package+/static/mobile/config.js_tmpl]: Fixed...

**Comment by elemoine** [comment]
Looks good to me.

**Pull request merged**